src/components/stories/Rating.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import { PageWrapper } from '../../core/storybook';
+import { Rating } from '..';
+
+export default {
+    title: 'Components/Rating',
+    component: Rating,
+};
+
+export const Intro = () => (
+    <PageWrapper>
+        <h1>Rating</h1>
+        <p>Wrapper of Material-UI Rating component.</p>
+    </PageWrapper>
+);
+
+export const Basic = () => {
+    const [value, setValue] = useState<number | null>(2);
+    return (
+        <PageWrapper>
+            <Rating
+                name="controlled"
+                value={value}
+                onChange={(event, newValue) => {
+                    setValue(newValue);
+                }}
+            />
+        </PageWrapper>
+    );
+};
+
+export const WithCustomIcon = () => {
+    return (
+        <PageWrapper>
+            <Rating name="with-custom-icon" icon={<FontAwesomeIcon icon={faHeart} />} defaultValue={2} />
+        </PageWrapper>
+    );
+};
+
+export const Precision = () => {
+    return (
+        <PageWrapper>
+            <Rating name="rating-with-precision" precision={0.5} defaultValue={2} />
+        </PageWrapper>
+    );
+};
+
+export const WithCustomCSS = () => {
+    return (
+        <PageWrapper>
+            <Rating
+                name="with-custom-css"
+                defaultValue={2}
+                css={`
+                    .MuiRating-label {
+                    font-size: 40px;
+                    }
+                    .MuiRating-iconFilled {
+                    color: #a00000};
+                    }
+                    .MuiRating-iconEmpty {
+                    color: #f8ecec;
+                    }`}
```

Comment:
Tiny readability adjustment?
```suggestion
                    .MuiRating-label {
                        font-size: 40px;
                    }
                    .MuiRating-iconFilled {
                        color: #a00000};
                    }
                    .MuiRating-iconEmpty {
                        color: #f8ecec;
                    }`}
```